### ISSUE TITLE
The value `touchEvent.locationX` equals NaN or INF and will be replaced by `0`.

### ISSUE THREAD

**Issue by amwebexpert** [JungHsuan/rn-gesture-flip-card#18]
💘 Very nice lib and thanks for sharing it! ❤️ 

Do you think it be possible to fix `console.warn` messages when reaching `90deg` while dragging?
Produced warning at `90deg`:
- `The value 'touchEvent.locationX' equals NaN or INF and will be replaced by '0'.`

Same apply also for `touchEvent.locationY`

**Comment by JungHsuan** [comment]
Yes! I believe that is possible. I will manage to fix this problem soon. Thanks for your comment.

**Comment by Stevemoretz** [comment]
> Yes! I believe that is possible. I will manage to fix this problem soon. Thanks for your comment.

Hi I have the same issue (not using your library) have you created an issue on reanimated yet?

**Comment by Andre-at-LaCapitale** [comment]
> Hi I have the same issue (not using your library) have you created an issue on reanimated yet?

@Stevemoretz this lib does _not_ uses `react-native-reanimated` dependency (it only depends directly on `react-native`)

**Comment by Stevemoretz** [comment]
> > Hi I have the same issue (not using your library) have you created an issue on reanimated yet?
> 
> @Stevemoretz this lib does _not_ uses `react-native-reanimated` dependency (it only depends directly on `react-native`)

Oh sorry! Sounds like a rn core issue then because I'm getting the exact problem on -90 or 90 with `react-native-reanimated`

**Comment by JungHsuan** [comment]
Sorry for my delay.
I've just published version 1.0.6 to fix this problem.
Please let me know if you have further questions.
Thanks!

**Comment by Stevemoretz** [comment]
Thank you ignoring the log is fine for now , but it's better to create an issue on Facebook  react native repository if there isn't one yet? I forgot to create one
